plugins/matplotlib/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const { resolve } = require('path');
+const webpack = require('webpack');
+
+module.exports = {
```

Comment:
The webpack configs could be shared probably. They look identical

Or use vite library mode?